packages/experiments/src/components/Stack/Stack.types.ts

```diff
@@ -80,15 +80,12 @@ export interface IStackProps
   grow?: boolean | number | 'inherit' | 'initial' | 'unset';
 
   /**
-   * Defines the horizontal spacing between Stack child elements.
+   * Defines the spacing between Stack children elements.
+   * The property is specified as a value for 'horizontal gap', followed optionally by a value for 'vertical gap'.
+   * If 'vertical gap' is omitted, it's set to the same value as 'horizontal gap'.
    */
   gap?: number | string;
```

Comment:
grid gap is "row gap then column gap"
![image](https://user-images.githubusercontent.com/1434956/51872092-22fc0700-230d-11e9-971b-1fb57b1cba6a.png)


so if we are calling it vertical and horizontal then it should be "vertical horizontal"

Reply:
actually i'm trying to figure out if you have this right or not. i think the issue is vertical/horizontal.....not sure if horizontal gap is gap between horizontal items, or gap running horizontally

can we switch to row and column gap?

Reply:
That's right. Hmmmm, do we want to have it as 'vertical horizontal' though? Doesn't matter that much to me, but given that we had the original `gap` as the horizontal one I want to raise the question.

CC'ing @dzearing to get his opinion.

Reply:
I 'think' the order is right, but naming convention is confusing. What is 'vertical gap'? distance between items stacked vertically...or the gap that runs vertically? I'm mostly wanting to make sure we are using the same approach as grid is, and suggesting we use their terminology when talking about the gap. Column gap vs horizontal/vertical.

Reply:
Fixed in latest commit 👍